ReadMe.md

```diff
@@ -41,7 +41,7 @@ Then you can fork the Labs repo, create a new branch, and start working on your
 ```ascii
 dotnet new --install .\template\ProjectTemplate\
 
-cd labs
+cd components
 
 dotnet new labexp -n MyExperimentNameHere
```

Comment:
In the future, we should change the template to be called 'toolkit-component' or something 'tkcmp'? not sure on naming restrictions for shortname on the dotnet templating tool.

Reply:
Sure, if we need to.